kubernetes/helm/startree-thirdeye/templates/coordinator/configmap.yaml

```diff
@@ -59,8 +59,12 @@ data:
 {{ toYaml .Values.auth | indent 6 }}
 
     database:
-      # Assuming a local MySQL server running on the default port 3306
+      # Defaults to k8s cluster local MySQL server running on port 3306, but uses URL if provided
+      {{ if .Values.mysql.mysqlUrl }}
```

Comment:
The condition should be on `.Values.mysql.enabled`. That will force user to provide the `mysqlUrl` when internal mysql is disabled. Also avoid the situation of thirdeye connecting to external mysql as well as spinning its own instance if `mysql.mysqlUrl` is also set and `mysql.enabled` is also set to `true`. Let me know your thoughts.